src/emulator/vpk.h

```diff
@@ -24,4 +24,4 @@ struct MemState;
 template <class T>
 class Ptr;
 
-bool load_vpk(Ptr<const void> &entry_point, IOState &io, MemState &mem, const std::wstring& path);
+bool load_vpk(Ptr<const void> &entry_point, IOState &io, MemState &mem, const std::wstring& path, std::string& game_title, std::string& title_id);
```

Comment:
I normally follow C standard library convention and put output parameters on the left.